baking/src/tezos_baking/tezos_setup_wizard.py

```diff
@@ -381,6 +381,30 @@ def get_key_mode_query(modes):
 )
 
 
+def get_stake_tez_query(staked_balance, minimal_frozen_stake):
+    def show_tez(tez):
+        # smallest tz quantity is microtez, so we are sure
+        # no digits after the 6th after floating point can occur
+        if int(tez) == 0:
+            return "0"
+        elif int(tez) % (10**6) == 0:
+            return tez[:-6] + tez[-6:-1].rstrip("0")
+        else:
+            return tez[:-6] + "." + tez[-6:-1].rstrip("0")
+
+    at_least = int(minimal_frozen_stake) - int(staked_balance)
+    return Step(
+        id="stake_tez",
+        prompt=f"In order to get baking rights, you need to stake at least {show_tez(minimal_frozen_stake)}Tz.\n"
```

Comment:
Since this rule is introduced by adaptive issuance and new staking mechanism, which is new for users and may be confusing, let's add a link to the documentation (e.g. [this one](https://tezos.gitlab.io/paris/adaptive_issuance.html#new-staking-mechanism)) here